src/assembler/remove_asm.cpp

```diff
@@ -15,6 +15,8 @@ Date:   December 2014
 
 #include "remove_asm.h"
 
+#include <ansi-c/c_expr.h>
```

Comment:
`src/assembler/module_dependencies.txt` does not currently list `ansi-c`, so we're introducing a new dependency on ansi-c here. And, really, this means that the expression is _not_ limited to the C front-end.